src/v/pandaproxy/schema_registry/seq_writer.cc

```diff
@@ -372,16 +386,18 @@ seq_writer::do_delete_subject_impermanent(
     }
 
     // Proceed to write
-    auto key = delete_subject_key{
-      .seq{write_at}, .node{seq._node_id}, .sub{sub}};
-    auto value = delete_subject_value{.sub{sub}};
-    auto batch = as_record_batch(key, value);
+    batch_builder rb{write_at, sub};
+    rb(
+      delete_subject_key{.seq{write_at}, .node{_node_id}, .sub{sub}},
+      delete_subject_value{.sub{sub}});
+
+    auto conf = co_await ss::coroutine::as_future(
+      _store.get_subject_config_written_at(sub));
+    if (!conf.failed()) {
+        rb(conf.get());
+    }
```

Comment:
is this a abandoned failed future waiting to happen? i think you need to eat the exception out of the future in the failed case.